assignments/closure.js

```diff
@@ -1,14 +1,29 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
+function nameIs(firstName, lastName){
+  var intro= "Your name is ";
+  function fullName(){
+    return intro + firstName + " " + lastName;
+  }
+  return fullName ();
+}
+nameIs ("Slim", "Shady");
 
 
 // ==== Challenge 2: Create a counter function ====
 const counter = () => {
   // Return a function that when invoked increments and returns a counter variable.
-};
+
+let counter = 0;
+let plusOne = () = >{
```

Comment:
When defining functions, you would want to use `const plusOne` rather than `let plusOne`. This is because functions generally will not be called again nor should they be altered.